src/AzureExtension/DataManager/AzureDataManagerUpdate.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using DevHomeAzureExtension.DataManager;
+using DevHomeAzureExtension.DataModel;
+using Microsoft.TeamFoundation.Policy.WebApi;
+using Serilog;
+
+namespace DevHomeAzureExtension;
+
+public partial class AzureDataManager
```

Comment:
Is this class supposed to be called AzureDataManager when we already have a class with that name?

Reply:
Yes, it's a partial class, which means its implementation can be spread across multiple files. I am attempting to reduce the size of the AzureDataManager from one enormous file to several conceptually-grouped files.